app/javascript/src/dropzone/bulk_submission_form_controller.js

```diff
@@ -0,0 +1,84 @@
+// Note on error handling:
+// We do not use dropzone options such as accept, acceptedFiles
+// and maxFilesize, preferring instead to use rails/ruby
+// validation to enforce these rules.Rails validation
+// errors are then returned as JSON. This provides consistent
+// JS-disabled and JS-enabled error messaging.
+
+// Dropzone errors, configurable using its options, are returned to the `error`
+// event via the response argument. These errors are simple STRINGs, whereas
+// the app returns the bulk_submission_form object errors JSON with an array for
+// the `uploaded_file`` attribute (specified by paramName). Therefore the
+// `error` event handles both these possible response types.
+//
+import Dropzone from 'dropzone'
+
+import {
+  addErrorMessage,
+  removeErrorMessages
+} from './govuk_errors_helper'
+
```

Comment:
This is a good idea separating out all the error stuff 👍  we should probably do this for the apply dropzone too